homeassistant/components/synology_srm/device_tracker.py

```diff
@@ -91,12 +99,8 @@ def _update_info(self):
         _LOGGER.debug("Scanning for connected devices")
 
         devices = self.client.core.network_nsm_device({"is_online": True})
-        last_results = []
-
-        for device in devices:
-            last_results.append({"mac": device["mac"], "hostname": device["hostname"]})
 
         _LOGGER.debug("Found %d device(s) connected to the router", len(devices))
 
-        self.last_results = last_results
+        self.last_results = devices
```

Comment:
Please rename `last_results` to `devices` to make it more descriptive. Furthermore, assign its value directly.

Reply:
I considered doing this at the time ... will do